src/Serilog.Sinks.AzureAnalytics/LoggerConfigurationExtensions.cs

```diff
@@ -57,8 +58,8 @@ public static class LoggerConfigurationExtensions
             LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
             bool storeTimestampInUtc = true,
             IFormatProvider formatProvider = null,
-            int logBufferSize = 2000,
-            int batchSize = 100,
+            int? logBufferSize = ConfigurationSettings.DefaultBufferSize,
+            int batchSize = ConfigurationSettings.DefaultBatchSize,
```

Comment:
Changed to use centralized constants and allow for passing null to preserve underlying default behavior of PeriodicBatchingSinkOptions